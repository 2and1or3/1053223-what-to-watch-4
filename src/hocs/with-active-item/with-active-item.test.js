import React from "react";
import renderer from "react-test-renderer";
import PropTypes from "prop-types";

import withActiveItem from './with-active-item.js';

const MockComponent = (props) => {
  const {activeItem} = props;

  return (<div activeItem={activeItem}/>);
};

MockComponent.propTypes = {
  activeItem: PropTypes.string.isRequired,
};

const MockComponentWithActiveItem = withActiveItem(MockComponent);

it(`Render component with active item`, () => {
  const tree = renderer
    .create(<MockComponentWithActiveItem />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});

// npm run test.jest -- hocs/with-active-item/with-active-item.test.js
