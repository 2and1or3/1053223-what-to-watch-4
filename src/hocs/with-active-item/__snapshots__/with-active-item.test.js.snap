// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render component with active item 1`] = `
<div
  activeItem=""
/>
`;
